### ISSUE TITLE
fedora 可以使用吗

### ISSUE THREAD

**Issue by tanchy82** [huan/docker-wxwork#1]
fedora 可以使用吗

**Comment by huan** [comment]
I believe it should be ok.

Please let me know the result if you tried on Fedora, thanks!

**Comment by tanchy82** [comment]
![2020-02-29 22-40-00屏幕截图](https://user-images.githubusercontent.com/19941245/75609541-7e1db000-5b44-11ea-8558-17cc499ef7b5.png)
I don't know is my docker problem or other problem? Do have hlep me? thanks!

**Comment by huan** [comment]
Yes, it should be your docker problem according to your screen shot.

**Comment by richoon-ke** [comment]
我使用fedora35,看起来工作的并不正常，没有任何错误信息，仅仅停留在启动后没有提示及二维码窗口

🚀 Pulling the latest docker image done.
🚀 Starting DoWork /dɑɑˈwɜːk/ ...

后没有任何有用信息
